### DISCUSSION TITLE
Release date for version 7?

### DISCUSSION THREAD

**Discussion by aterbo** [Blazored/Modal#426] [Q&A] [answered]
Hi @chrissainty 

I'm curious if you have an idea when v 7.0 will be formally released to NuGet? I'm eager to use some of the included features, but I'd prefer not to use a preview package.

Not trying to rush. Just curious more than anything.

**Comment by chrissainty** [comment, marked as answer]
Hey @aterbo, I'm just working on a new docs site. It's the last thing and then v7 can go live. If it helps you, there won't be any more changes in v7 so there is minimal risk in using the preview.

**Reply by aterbo** [reply]
Great, thanks @chrissainty

As always, this library is greatly appreciated.
